cheroot/ssl/builtin.py

```diff
@@ -52,6 +52,36 @@ def _assert_ssl_exc_contains(exc, *msgs):
     return any(m.lower() in err_msg_lower for m in msgs)
 
 
+def _loopback_for_cert_thread(context, server):
+    """Wrap a socket in ssl and perform the server-side handshake."""
+    # As we only care about parsing the certificate, the failure of
+    # which will cause an exception in ``_loopback_for_cert``,
+    # we can safely ignore connection and ssl related exceptions.
```

Comment:
I think, adding a link here would be useful:
```suggestion
    # we can safely ignore connection and ssl related exceptions. Ref:
    # https://github.com/cherrypy/cheroot/issues/302#issuecomment-662592030
```